superset/views/dashboard/views.py

```diff
@@ -40,6 +40,8 @@
 from ..utils import bootstrap_user_data
 from .mixin import DashboardMixin
 
+config = app.config
```

Comment:
is there a reason why we don't reference the app from inside the function instead of pulling the config out at global scope? I know i've seen this pattern a lot, but we've run into issues with tests because of it in the past. do you think it might be better to move it to the function? cc @john-bodley for his thoughts

Reply:
I think this is used for connivence though in general I’m not certain the abstraction is worth it. I would prefer we use `app.config[...]` on line #62.

Reply:
was just following the first pattern I saw, makes no difference to me.